tests/functional/views/admin_test.py

```diff
@@ -0,0 +1,5 @@
+def test_admin_authentication_redirects_to_google(app):
+    response = app.get("/admin/instances/")
+
+    assert response.status_code == 302
+    assert response.location.startswith("http://localhost/googleauth/login")
```

Comment:
I think you might be able to use `h-matchers` to do this in a single `assert`: https://github.com/hypothesis/h-matchers/blob/main/docs/matching-web.md